codec-redis/src/main/java/io/netty/handler/codec/redis/AbstractCollectionRedisMessage.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.util.AbstractReferenceCounted;
+import io.netty.util.ReferenceCountUtil;
+import io.netty.util.internal.ObjectUtil;
+import io.netty.util.internal.StringUtil;
+import io.netty.util.internal.UnstableApi;
+
+import java.util.Collection;
+
+/**
+ * Abstract class for Aggregate data types message.
+ */
+@UnstableApi
+public abstract class AbstractCollectionRedisMessage extends AbstractReferenceCounted
+    implements RedisMessage, AggregatedRedisMessage {
+
+    protected Collection<RedisMessage> children;
+
+    public AbstractCollectionRedisMessage() {
+    }
+
+    public AbstractCollectionRedisMessage(Collection<RedisMessage> children) {
+        this.children = ObjectUtil.checkNotNull(children, "children");
+    }
+
+    /**
+     *
+     * @return
+     */
+    public abstract Collection<RedisMessage> children();
+
+    @Override
+    protected void deallocate() {
+        for (RedisMessage msg : children) {
```

Comment:
this will throw if `children` is null

Reply:
I had remove unused default constructor 
`
  public AbstractCollectionRedisMessage() {
  }
`
so, `children` will not be null?